wallet/storage/src/schema.rs

```diff
@@ -53,5 +54,7 @@ storage::decl_schema! {
         pub DBUnconfirmedTxCounters: Map<AccountId, u64>,
         /// Store for each account's legacy VRF public key
         pub DBVRFPublicKeys: Map<AccountId, AccountVrfKeys>,
+        /// Store for separate keys added to accounts
+        pub DBSeparateKeys: Map<AccountAddress, AccountSeparateKey>,
```

Comment:
I see the naming "separate" has infected the whole codebase... please rename them all to `standalone`. The word separate isn't really the best at describing this.